evaluation/base/semantic.pairs.txt

```diff
@@ -1,44 +1,66 @@
+: Politiker,Partei
+Merkel,CDU
+Schröder,SPD
+Seehofer,CSU
+Westerwelle,FDP
+Wagenknecht,Linkspartei
+Gysi,PDS
```

Comment:
He was part of both parties (in addition PDS is part of Linke). It seemed for me that he is more connected with PDS than Linke, because he was chairman of PDS